lib/myfinance/resources/receivable_account.rb

```diff
@@ -14,12 +14,19 @@ def receive(id, entity_id, params = {})
         end
       end
 
+      def undo_receivement(id, entity_id)
+        http.put(endpoints(id, entity_id, :undo_receivement)) do |response|
+          respond_with_object(response, 'receivable_account')
+        end
+      end
+
       private
 
       def endpoints(id, entity_id, key)
         {
           create: '/entities/:entity_id/receivable_accounts',
-          receive: '/entities/:entity_id/receivable_accounts/:id/receive'
+          receive: '/entities/:entity_id/receivable_accounts/:id/receive',
+          undo_receivement: '/entities/:entity_id/receivable_accounts/:id/undo_receivement'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.